packages/@sanity/base/src/components/rovingFocus/useRovingFocus.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import {useCallback, useEffect, useState} from 'react'
+import {RovingFocusProps} from '.'
+
+const EMPTY_ARRAY = []
+
+const MUTATION_ATTRIBUTE_FILTER = ['aria-hidden', 'disabled', 'href']
+
+const FOCUSABLE =
+  'a[href], button, input, textarea, select, details, [tabindex]:not([tabindex="-1"])'
+
+function getFocusableElements(element: HTMLElement) {
+  return [...(element.querySelectorAll(FOCUSABLE) as any)].filter(
+    (el) => !el.hasAttribute('disabled') && !el.getAttribute('aria-hidden="true"')
```

Comment:
did you mean `el.getAttribute('aria-hidden') !== 'true'`? i'm not familiar with that usage

Reply:
Your suggestion is better – will update with `el.getAttribute('aria-hidden') !== 'true'` ! 👍